share/jupyterhub/templates/spawn_pending.html

```diff
@@ -17,7 +17,12 @@
             <span class="sr-only"><span id="sr-progress">0%</span> Complete</span>
           </div>
         </div>
-        <p id="progress-message"></p>
+        <p id="progress-message">Server requested</p>
```

Comment:
This should not be changed. This field should only contain messages that have come from the server. If the tests fail as a result of the changes here, that might mean a problem in the changes or that the test needs some updating.